inc/class-upgrade.php

```diff
@@ -326,9 +326,12 @@ private function upgrade_49() {
 			$wpdb->prepare('
 				SELECT user_id, meta_value
 				FROM ' . $wpdb->usermeta . '
-				WHERE meta_key = %s AND meta_value LIKE "%%wpseo-dismiss-about%%"
-				', $meta_key ),
-				ARRAY_A
+				WHERE meta_key = %s AND meta_value LIKE %s
+				',
+				$meta_key,
+				'%%wpseo-dismiss-about%%'
```

Comment:
As this is no longer in the prepare string itself, the double `%%` should be single `%`, as they don't need to be escaped anymore.

Reply:
Oops.. Thanks for catching that :blush: Fixed now (amended commit).